docs/guides/deleting-clusters-with-removed-worker-pools.md

```diff
@@ -0,0 +1,97 @@
+# Deleting ROSA Clusters with Removed Worker Pools
+
+When managing ROSA clusters with Terraform, you may encounter situations where the initial worker pools (`workers-[0-n]`) have been deleted, but you still need to delete the cluster. This guide explains how to use the `ignore_deletion_error` field to handle this scenario.
+
+## Overview
+
+The `ignore_deletion_error` field is available on both HCP and Classic machine pool resources. It allows Terraform to remove the machine pool resource from its state while ignoring API errors that occur during the deletion process.
```

Comment:
please include why the situation occurs, shed light on what this is solving for.